gen/dcos-config.yaml

```diff
@@ -211,9 +211,57 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
+  - path: /etc_master/logrotate_master.config
+    content: |
+      compress
+      delaycompress
+      notifempty
+      missingok
+      nomail
+      /var/lib/dcos/mesos/log/* {
+          olddir /var/lib/dcos/mesos/log/archive
+          maxsize 256M
+          rotate {{ mesos_log_retention_count }}
+          copytruncate
+          postrotate
+              find /var/lib/dcos/mesos/log/ /var/lib/dcos/mesos/log/archive -mtime +14 -delete
+          endscript
+      }
+  - path: /etc_slave/logrotate_agent.config
```

Comment:
These are identical, why not just write one to `/etc/logrotate_agent.config` and then just make the unit point to that one?